internal/compiler-bridge/src/main/scala/xsbt/ExtractUsedNames.scala

```diff
@@ -49,29 +52,39 @@ import Compat._
 class ExtractUsedNames[GlobalType <: CallbackGlobal](val global: GlobalType) extends Compat with ClassName with GlobalHelpers {
   import global._
 
-  def extract(unit: CompilationUnit): Iterable[(String, Iterable[String])] = {
+  def extract(unit: CompilationUnit): HashMap[String, ArrayList[String]] = {
```

Comment:
I had to fall back to using concrete types because I was unable to make this work with Java's `forEach`